kia-lib/src/test/kotlin/com/faforever/ice/IceAdapterIT.kt

```diff
@@ -47,6 +48,9 @@ class IceAdapterIT {
     @MockK
     private lateinit var apiClientMock2: ApiClient
 
+    @MockK(relaxed = true)
```

Comment:
Relaxed because `telemetryClient.updateCoturnList()` on this mock gets called when `IceAdapter` starts up.  I could just provide mock return value for this function in the `BeforeEach` which would be more precise, but I expect there will be more `TelemetryClient` functions that get called from these integration tests. In this case at least, it wasn't clear to me why this test was failing after adding `telemetryClientMock`. The error seemed unrelated: `Connection refused: connect` from `FakeGameClient`.  The actual problem: calling the unimplemented mock function didn't appear because it happens on another thread so the exception wasn't logged (I think). Making this mock relaxed reduces the chance of this confusion in the future. But, maybe there's a better way.